monitor/reporter.go

```diff
@@ -2,6 +2,7 @@ package monitor
 
 import "github.com/influxdata/influxdb/models"
 
+// Reporter ...
```

Comment:
This needs a proper comment. Perhaps something like: `Reporter gathers internal statistics.`